src/components/ebay-button/README.md

```diff
@@ -1,12 +1,15 @@
 # ebay-button
 
-## Usage
+## ebay-button Tag
```

Comment:
Is the repetition needed for the nested tag case, and therefore applied to all cases (even components that are singular)?

Reply:
Yes, I agree. The repetition of `ebay-button` seems redundant

Reply:
Because headings get an `id` automatically generated for them (to create and easy table of contents with anchors) each one needs to be unique.

Reply:
Is it certain we would actually use a TOC like https://github.com/thlorenz/doctoc? And if we might, shouldn't we defer this until that decision is made? The repetition and verbosity does seem a bit much, especially for cases like the `button`, which don't need it.

Reply:
We are planning for the future right now. And, it’s not entirely about a TOC. I use IDs all the time to send people like to specific parts of a document. And, if the component docs were ever in a single page (not advocating for this) they would all be navigable.

Also, the _pattern_ is what we want to establish for everyone as we contribute, and accept outside contribution. Some components have more than one tag, where some don’t. I guess I’m trying to accommodate both scenarios.

Lastly, I have been using a Markdown linter in VS code, which has helped identify possible problem areas for the broad swathe of MD parsers which are out there.

All that said, I will look at how to make it less verbose and repetitive.